services/install-unifi.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+function install {
+  # create service directory
+  mkdir -p /srv/unifi
+
+  # create yml(s)
+  {
+    echo "version: '2.1'"
+    echo "services:"
+    echo "  unifi-controller:"
+    echo "    image: linuxserver/unifi-controller"
+    echo "    environment:"
+    echo "      - PUID=\${PUID}"
+    echo "      - PGID=\${PGID}"
+    echo "      - MEM_LIMIT=\${MEM_LIMIT}"
+    echo "    ports:"
+    echo "      - 3478:3478/udp"
+    echo "      - 10001:10001/udp"
+    echo "      - 8080:8080"
+    echo "      - 8081:8081"
+    echo "      - 8443:8443"
+    echo "      - 8843:8843"
+    echo "      - 8880:8880"
+    echo "      - 6789:6789"
+    echo "    volumes:"
+    echo "      - \"/srv/unifi:/root/.unifi-controller\""
+    echo "    restart: unless-stopped"
+  } > /srv/unifi/unifi.yml
+
+  # create .env with default values
+  {
+    echo "PUID=1000"
+    echo "PGID=1000"
+    echo "MEM_LIMIT=1024M"
+  } > /srv/unifi/.env
+
+  # add autorun
+  {
+    echo "unifi_autorun=true"
+    echo
+    echo "if [ \"\$unifi_autorun\" = true ]; then"
+    echo "  treehouses services unifi up"
+    echo "fi"
+    echo
+    echo
+  } > /srv/unifi/autorun
+}
+
+# environment var
+function uses_env {
+  echo true
+}
+
+# add supported arm(s)
+function supported_arms {
+  echo "v7l"
+  echo "v6l"
+}
+
+# add port(s)
+function get_ports {
+  echo "8443 use https"
```

Comment:
Remove the `use https`

Reply:
Needs https to get to the web gui though? well anyway.